percona-xtrabackup-8.4 FTBFS: sphinx.errors.ExtensionError: Could not import extension sphinx.builders.latex

Possibly unrelated, but sphinx was bumped to a new upstream version yesterday.

```
2025/02/19 17:51:50 INFO [  5%] Building C object extra/libkmip/libkmip/src/CMakeFiles/demo_create.dir/demo_create.c.o
2025/02/19 17:51:51 INFO Running Sphinx v8.2.0
2025/02/19 17:51:51 INFO loading translations [en]... done
2025/02/19 17:51:51 INFO [  5%] Linking C executable demo_create
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN Extension error!
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN Versions
2025/02/19 17:51:51 WARN ========
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN * Platform:         linux; (Linux-6.8.0-1021-gcp-x86_64-with-glibc2.40)
2025/02/19 17:51:51 WARN * Python version:   3.13.2 (CPython)
2025/02/19 17:51:51 WARN * Sphinx version:   8.2.0
2025/02/19 17:51:51 WARN * Docutils version: 0.21.2
2025/02/19 17:51:51 WARN * Jinja2 version:   3.1.5
2025/02/19 17:51:51 WARN * Pygments version: 2.19.1
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN Last Messages
2025/02/19 17:51:51 WARN =============
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN None.
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN Loaded Extensions
2025/02/19 17:51:51 WARN =================
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN None.
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN Traceback
2025/02/19 17:51:51 WARN =========
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN       File "/usr/lib/python3.13/site-packages/sphinx/registry.py", line 544, in load_extension
2025/02/19 17:51:51 WARN         raise ExtensionError(
2025/02/19 17:51:51 WARN             __('Could not import extension %s') % extname, err
2025/02/19 17:51:51 WARN         ) from err
2025/02/19 17:51:51 WARN     sphinx.errors.ExtensionError: Could not import extension sphinx.builders.latex (exception: No module named 'roman_numerals')
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN The full traceback has been saved in:
2025/02/19 17:51:51 WARN /tmp/sphinx-err-4psnx4fk.log
2025/02/19 17:51:51 WARN
2025/02/19 17:51:51 WARN To report this error to the developers, please open an issue at <https://github.com/sphinx-doc/sphinx/issues/>. Thanks!
2025/02/19 17:51:51 WARN Please also report this if it was a user error, so that a better error message can be provided next time.
2025/02/19 17:51:51 WARN make[2]: *** [storage/innobase/xtrabackup/doc/source/CMakeFiles/man.dir/build.make:73: storage/innobase/xtrabackup/doc/source/build/man] Error 2
2025/02/19 17:51:51 WARN make[1]: *** [CMakeFiles/Makefile2:3480: storage/innobase/xtrabackup/doc/source/CMakeFiles/man.dir/all] Error 2
```

--- a/py3-roman-numerals-py.yaml
+++ b/py3-roman-numerals-py.yaml
@@ -1,0 +1,97 @@
+package:
+  name: py3-roman-numerals-py
+  version: 3.0.0
+  epoch: 0
+  description: A library to convert between integers and Unicode Roman numerals
+  copyright:
+    - license: CC0-1.0 AND 0BSD
+  dependencies:
+    provider-priority: "0"
+
+data:
+  - name: py-versions
+    items:
+      "3.10": "310"
+      "3.11": "311"
+      "3.12": "312"
+      "3.13": "313"
+
+vars:
+  import: roman_numerals
+  pypi-package: roman-numerals-py
+
+environment:
+  contents:
+    packages:
+      - build-base
+      - busybox
+      - ca-certificates-bundle
+      - py3-supported-build-base
+      - py3-supported-flit-core
+      - wolfi-base
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/AA-Turner/roman-numerals
+      expected-commit: 6727a2c61cf03026c2ae76f12e01485604a6a8d9
+      tag: v${{package.version}}
+
+subpackages:
+  - range: py-versions
+    name: py${{range.key}}-${{vars.pypi-package}}
+    pipeline:
+      - runs: cp LICENCE.rst python/
+      - name: Python Build
+        uses: py/pip-build-install
+        with:
+          python: python${{range.key}}
+        working-directory: ./python
+    dependencies:
+      provider-priority: ${{range.value}}
+    test:
+      environment: {}
+      pipeline:
+        - name: Import Test
+          uses: python/import
+          with:
+            import: ${{vars.import}}
+            python: python${{range.key}}
+
+  - name: py3-supported-${{vars.pypi-package}}
+    description: meta package providing ${{vars.pypi-package}} for supported python versions.
+    dependencies:
+      runtime:
+        - py3.10-${{vars.pypi-package}}
+        - py3.11-${{vars.pypi-package}}
+        - py3.12-${{vars.pypi-package}}
+        - py3.13-${{vars.pypi-package}}
+    test:
+      pipeline:
+        - uses: python/import
+          with:
+            python: python3.10
+            import: ${{vars.import}}
+        - uses: python/import
+          with:
+            python: python3.11
+            import: ${{vars.import}}
+        - uses: python/import
+          with:
+            python: python3.12
+            import: ${{vars.import}}
+        - uses: python/import
+          with:
+            python: python3.13
+            import: ${{vars.import}}
+
+test:
+  pipeline:
+    - uses: test/emptypackage
+
+update:
+  enabled: true
+  github:
+    identifier: AA-Turner/roman-numerals
+    strip-prefix: v
+    use-tag: true

--- a/py3-sphinx.yaml
+++ b/py3-sphinx.yaml
@@ -1,7 +1,7 @@
 package:
   name: py3-sphinx
   version: "8.2.0"
-  epoch: 0
+  epoch: 1
   description: Python Documentation Generator
   copyright:
     - license: MIT
@@ -48,6 +48,7 @@ subpackages:
         - py${{range.key}}-packaging
         - py${{range.key}}-pygments
         - py${{range.key}}-requests
+        - py${{range.key}}-roman-numerals-py
         - py${{range.key}}-snowballstemmer
         - py${{range.key}}-sphinxcontrib-applehelp
         - py${{range.key}}-sphinxcontrib-devhelp
@@ -72,6 +73,7 @@ subpackages:
             python: python${{range.key}}
             imports: |
               import ${{vars.import}}
+              from sphinx.writers.latex import LaTeXTranslator, LaTeXWriter
 
   - range: py-versions
     name: py${{range.key}}-${{vars.pypi-package}}-bin

